oat-sa/extension-tao-testqti
# Fix/tao 9435/when paused, end timer only if target client

**Related to tasks:** 
- https://oat-sa.atlassian.net/browse/TAO-9435 
- https://oat-sa.atlassian.net/browse/NCC-176

**Description:** 
A bug was introduced in https://github.com/oat-sa/extension-tao-testqti/pull/1644/files
The bug was that timers no longer keep ticking once a security pause has occurred
That code was recently reverted in https://github.com/oat-sa/extension-tao-testqti/pull/1724 because we needed to update Nccer and not face that bug. Now, this PR fixes the bug by checking `timer target` in configuration, because we can't stop BE timers and we can stop FE sided timers. 

**How to test:** 
- Start a proctored test with security plugins enabled
- Enter a timed section
- Ctrl+TAB
- you are paused, wait a bit
- resume
- as a proctor, allow the resume
- as a TT, check the timer, it kept ticking as if it wasn't paused

**Steps to reproduce:** 
- Start a proctored test with security plugins enabled
- Enter a timed section
- Ctrl+TAB
- you are paused, wait a bit
- resume
- as a proctor, allow the resume
- as a TT, check the timer, it's picked up where it's left
